src/DendroDocs.Tool/Analyzers/InvocationsAnalyzer.cs

```diff
@@ -71,23 +71,14 @@ public override void VisitInvocationExpression(InvocationExpressionSyntax node)
             return;
         }
 
-        var containingType = semanticModel.GetSymbolInfo(expression).Symbol?.ContainingSymbol.ToDisplayString();
-        if (containingType == null)
-        {
-            containingType = semanticModel.GetSymbolInfo(expression).CandidateSymbols.FirstOrDefault()?.ContainingSymbol.ToDisplayString();
-        }
+        var containingType = semanticModel.GetSymbolInfo(expression).Symbol?.ContainingSymbol.ToDisplayString() ?? semanticModel.GetSymbolInfo(expression).CandidateSymbols.FirstOrDefault()?.ContainingSymbol.ToDisplayString() ?? string.Empty;
```

Comment:
This line becomes very long (that's why it wasn't using the coalesce operator).
Maybe put this in a method, or have the symbol resolving and the `ToDisplayString` in two steps?

Reply:
Fixed it in the way I thought most suitable, let me know what you think